test/e2e/config/e2e_conf.yaml

```diff
@@ -89,7 +89,7 @@ intervals:
   default/wait-delete-cluster: ["20m", "10s"]
   default/wait-machine-upgrade: ["50m", "10s"]
   default/wait-machine-remediation: ["30m", "10s"]
-  default/wait-vm-state: ["10m", "100ms"]
+  default/wait-vm-state: ["20m", "100ms"]
```

Comment:
If I am not wrong, this change is for the bug in remediation reported by community, right? It is fixed in https://github.com/metal3-io/cluster-api-provider-metal3/pull/420. So maybe we can keep the time to 10m

Reply:
It was partly but I left it there because I have seen it needing a bit more time when I was testing on laptop.